reconcile/test/fixtures/jinja2/expected_result.yaml

```diff
@@ -0,0 +1,15 @@
+groups:
+- name: test-group
+  rules:
+  - record: test_metric
+    expr: vector(1)
+  - alert: TestAlert
+    expr: |
+      sum(rate(http_requests_total{status=~"5.."}[5m]))
+      /
+      sum(rate(http_requests_total[5m]))
+    labels:
+      service: app-interface
+      severity: critical
+    annotations:
+      summary: Test summary
```

Comment:
can you check if actually generated yaml has a newline at the end of file?

Reply:
Confirmed newline exists. Updated fixture to include [94795fc](https://github.com/app-sre/qontract-reconcile/pull/5153/commits/94795fc3c2efd3775e1c0edde24d0b894987c8d6)